docs/partitions.md

```diff
@@ -1,24 +1,24 @@
 # Partitions
 
 In the original implementation of ringpop, if a cluster is split to multiple
-partitions, nodes in each partition declare each other as faulty, and don't
-communicate. Ringpop implemented support for merging the partitions, which we
-call `healing`.
+partitions, nodes in each partition declare each other as faulty, and afterward
+will no longer communicate. Ringpop implemented support for merging the
+partitions, which we call `healing`.
 
-## Introdction -- basic algorithm
+## Introduction -- basic algorithm
 
 In order for two partitions to heal, the algorithm does the following,
-periodically (number of times per time unit is upper bounded):
+periodically:
```

Comment:
We could clarify that some important details are omitted in favour of brevity/clarity and they can be found in the code/documentation